README.md

```diff
@@ -7,14 +7,34 @@
    <a href="https://github.com/typeable/octopod/actions?query=workflow%3ADocumentation+branch%3Amaster"><img src="https://github.com/typeable/octopod/workflows/Documentation/badge.svg?branch=master" alt="Documentation"></a>
 </p>
 
-_Octopod_ is a fully open-source self-hosted solution for managing multiple deployments in a _Kubernetes_ cluster with a user-friendly web interface. Managing deployments does not require any technical expertise.
+_Octopod_ is a fully open-source self-service portal which empowers your team to leverage the full power of Kubernetes without the need for technical expertise.
 
-We created _Octopod_ because we believe that everything we release should be rigorously tested, however, such desires greatly [complicate the development workflow](docs/en/PM_case_study.md) leading to longer release cycles. We use _Octopod_ to mitigate the downsides of rigorously testing each feature by deploying every single change we make to a separate staging environment allowing QA to investigate each feature independently and in parallel.
+Octopod is a way for you to provide a simple interface for managing instances of a service in your Kubernetes cluster. Anything you can describe in a Helm Chart, any member of team can deploy. No matter how complex the deployment process is.
+
+The recipe is simple: you provide a url to your Helm chart, Octopod then automatically extracts the possible configuration that the Chart can have, and automatically manages the lifecycle of the services. What users see is just a simple web interface:
 
 ## 🖥 Demo
 
 <p align="center"><img src="img/demo.gif"></img></p>
 
+## 🧑‍🔬 Try it out for yourself
+
+You can have the installation from the demo above running on your local machine in minutes by running the following command:
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/typeable/octopod/master/octopod_local_install.sh)"
+```
+
+You can dive deeper into the installation process in our [deployment guide](docs/en/Octopod_deployment_guide.md).
```

Comment:
Maybe we should direct user to the doc which describes how install charts with Octopod (I mean Helm-based_deployment_guide.md)? I think it would be nice after we provided the sctipt

Reply:
I thought of this more as "if you want to read more about what this command does"

Reply:
+ we have a link to the helm-based deployment guide at the end of the linked doc